build/Targets/VSL.Settings.targets

```diff
@@ -75,9 +75,8 @@
     <MicrosoftCodeAnalysisAnalyzersVersionNext Condition="'$(MicrosoftCodeAnalysisAnalyzersVersionNext)' == ''">1.2.0</MicrosoftCodeAnalysisAnalyzersVersionNext>
 
     <!-- If we're on Visual Studio 2015 RTM and  we still have Roslyn installed, our VSIX-producing packages are not going to install, so let's not even try.
-         Update 1 supports this, so we'll use csi.exe's existence as a simple proxy for whether we have Update 1 installed or not.
-         OpenSourceDebug overrides this by simple virtue that it's property setting is after the include of this file. -->
-    <DeployExtension Condition="'$(DeployExtension)' == '' AND Exists('$(DevEnvDir)\CommonExtensions\Microsoft\Roslyn\Language Services\extension.vsixmanifest') AND !Exists('$(MSBuildBinPath)\csi.exe')">false</DeployExtension>
+         Update 1 supports this, so we'll use csi.exe's existence as a simple proxy for whether we have Update 1 installed or not. -->
+    <DeployExtension Condition="Exists('$(DevEnvDir)\CommonExtensions\Microsoft\Roslyn\Language Services\extension.vsixmanifest') and !Exists('$(MSBuildBinPath)\csi.exe')">false</DeployExtension>
```

Comment:
I believe the assumption was once Update 1 ships, we'll mandate Update 1 for building Roslyn. At that point, we can just delete this magic entire. The only reason it existed was to avoid the VS SDK errors when you tried building on RTM and hadn't ripped out the language services manually.